.github/workflows/ci.yml

```diff
@@ -0,0 +1,48 @@
+name: Test IPython CDI profile
+
+on:
+  push:
+  pull_request:
+  workflow_dispatch:
+  workflow_call:
+    inputs:
+      version: # the variable you can use in place of a matrix
+        required: true
+        type: string
+
+jobs:
+  testing_beamline_profiles:
+    name: ${{ matrix.repos.beamline-acronym }}-py${{ matrix.zenodo.python }}
```

Comment:
We should also add the `needs:` key as here: https://github.com/NSLS2/nsls2-collection-tiled/blob/acb874e327d6c659700d1df6bd06d00523dc9b3a/.github/workflows/package.yml#L33